lib/please_cli/please_cli/config.py

```diff
@@ -288,11 +310,15 @@
                 'heroku_app': 'releng-staging-treestatus',
                 'heroku_dyno_type': 'web',
                 'url': 'https://treestatus.staging.mozilla-releng.net',
+                # TODO: we need to change this to SSL Endpoint
```

Comment:
@dividehex can we switch to SSL Endpoint for ``releng-staging-treestatus`` heroku app?